benchmarking/nursery/lstm_wikitext2/lstm_wikitext2.py

```diff
@@ -22,10 +22,10 @@
 from syne_tune.report import Reporter
```

Comment:
Is there a reason for moving this one in `nursery/` (as opposed to distilbert which is in `training_script/`?)

Reply:
In the long run I think we can deprecate this benchmark and replace it by something more state-of-the-art (e.g based on transformers). But if you think it will remain relevant, we can have it in the main directory.

Reply:
Thanks makes sense either way is fine with me.